### DISCUSSION TITLE
How to convert LaTeX to Quarto via pandoc

### DISCUSSION THREAD

**Discussion by erikerhardt** [quarto-dev/quarto-cli#5933] [Q&A]
### Description

Is there a pandoc style that will correctly convert a LaTeX book to quarto, and not just generic markdown?  Is it related to the pandoc templates <https://quarto.org/docs/journals/templates.html>, and if so, how are they used?

I'm converting a LaTeX book to quarto, and I'm hoping to leverage some tools to do much of the conversion for me.

While the pandoc command below gets me pretty close, the markdown style is not the same as rmarkdown or quarto.  For example, Headings 1 (Chapter) and 2 (Section) use underlines.
pandoc -f latex -t latex.tex -o markdown.md
https://pandoc.org/MANUAL.html

Example input:
```
\chapter{Chapter}
\section{Section}
\subsection{Subsection}
\subsubsection{Subsubsection}
```

Example output:
```
Chapter
=======

Section
--------------

### Subsection
#### Subsubsection
```

**Comment by mcanouil** [comment]
Note that this question is a Pandoc question not really Quarto.

This being said, there are changes of syntaxes as you implied.
The output syntax you show is valid default markdown syntax.
Over the years, in rmarkdown, users started to only use `#` (myself included).
What you can try is look for variants of markdown, see <https://pandoc.org/MANUAL.html#markdown-variants>.

Note that, you can run the bundle version of Pandoc used in Quarto `quarto pandoc ...`

**Reply by erikerhardt** [reply]
Thank you, Mickaël.  Your recommended "quarto pandoc ..." command gets me much closer!
